rabbitmq_admin/tests/test_api.py

```diff
@@ -57,26 +40,29 @@ def setUpClass(cls):
             routing_key='test_queue',
             body='Test Message')
 
+        url = 'http://{host}:{port}'.format(host=cls.host,
+                                            port=cls.admin_port)
+        cls.api = AdminAPI(url, auth=('guest', 'guest'))
+
+        # connection statistics appear with a delay therefore:
+        time.sleep(5)
+
+        cls.node_name = cls.api.list_nodes()[0]['name']
+
     @classmethod
     def tearDownClass(cls):
         cls.connection.close()
 
     def setUp(self):
```

Comment:
You can completely remove this method

Reply:
Or you can try adding rabbit cleansing here

Reply:
Thanks! Removed.